### PR TITLE
various fixes to the systemd-boot documentation

### PR DESCRIPTION
**Pull request by poettering**
Mostly fix-ups for the BLS merge. Only documentation changes, no code.

### CODE REVIEW

File: doc/BOOT_LOADER_SPECIFICATION.md [hunk 1]
```diff
@@ -133,8 +133,7 @@ There are a couple of items that are out of focus for this specifications:
 
 ## Links
 
-<strike>http://www.freedesktop.org/wiki/Software/gummiboot/</strike><br>
```
Comment (keszybz) [inline doc/BOOT_LOADER_SPECIFICATION.md:134]: The gummiboot link is now 404, so there is no point in keeping it.

File: man/systemd-boot.xml [hunk 2]
```diff
@@ -25,47 +25,50 @@
...
+    runtime. Configuration file fragments, kernels, initrds and other EFI images to boot generally need to reside on
+    the ESP. Linux kernels must be built with <option>CONFIG_EFI_STUB</option> to be able to be directly executed as an
+    EFI image. During boot systemd-boot automatically assembles a list of boot entries from the following
+    sources:</para>
+
+    <itemizedlist>
+      <listitem><para>Boot entries defined with <ulink
+      url="https://github.com/systemd/systemd/blob/master/doc/BOOT_LOADER_SPECIFICATION.md">Boot Loader
+      Specification</ulink> description files located in <filename>/loader/entries/</filename> on the ESP. These
+      usually describe Linux kernel images with associated initrd images, but alternatively may also described
```
Comment (keszybz) [inline man/systemd-boot.xml]: described → describe

File: man/systemd-boot.xml [hunk 3]
```diff
@@ -25,47 +25,50 @@
...
+      arbitrary other EFI executables.</para></listitem>
+
+      <listitem><para>Unified kernel images following the <ulink
+      url="https://github.com/systemd/systemd/blob/master/doc/BOOT_LOADER_SPECIFICATION.md">Boot Loader
+      Specification</ulink> recommendations, as executable EFI binaries in <filename>/EFI/Linux/</filename> on the
```
Comment (keszybz) [inline man/systemd-boot.xml]: <strike> recommendations</strike>

**Comment by poettering** [comment]
I force pushed a new version. Only two changes are the ones @keszybz pointed out. PTAL

**Pull request merged**